README.md

```diff
@@ -83,6 +83,19 @@ of options:
  * [Libvirt][libvirt]
  * [Cockpit][cockpit]
 
+## Submitting patches
+
+When sending patches to the project, the submitter is required to certify that
+they have the legal right to submit the code. This is achieved by adding a line
+
+    Signed-off-by: Real Name <email@address.com>
```

Comment:
Right they encourage you, but when you read on, and that applies to both CLA and DCO, the author writes:

> However, most Conservancy projects use the same time-honored and successful mechanism used throughout the 35 year history of the Free Software community. Simply, they publish clearly in their developer documentation and/or other key places (such as mailing list subscription notices) that submissions [...] indicate the contributors' assent for inclusion of that software in the canonical version under the project's license

The other link explicitly names the DCO, as a simple alternative to a CLA, in case a lawyer only insists on explicit signing. To quote their scenario:

> Your lawyers want all contributors to expressly accept (sign, online or offline) contribution terms, perhaps because they feel the open source license itself is not enough (even though it is!). If this is the only concern, a contributor agreement that affirms the project’s open source license should be enough. The jQuery Individual Contributor License Agreement is a good example of a lightweight additional contributor agreement. For some projects, a Developer Certificate of Origin can be an even simpler alternative.

So again, no one there says that it is needed.